addons/l10n_tr/models/exception_reason.py

```diff
@@ -0,0 +1,19 @@
+from odoo import api, fields, models
+
+
+class ExceptionReason(models.Model):
+    _name = 'l10n_tr.exception_reason'
+    _description = 'Tax exemption/withdrawal reason'
+
+    name = fields.Char()
+    code = fields.Char(size=3)
+    display_name = fields.Char(compute='_compute_display_name')
```

Comment:
by default, the ORM is already adding an extra field called `display_name` that contains the result of `name_get`. So, you can drop this field.